_includes/meta.html

```diff
@@ -54,11 +54,10 @@
   <link rel="apple-touch-icon-precomposed" sizes="144x144" href="{{site.baseurl}}/assets/uswds/img/favicons/favicon-144.png">
   <!-- CSS
     ================================================== -->
-  <!-- env: demo -->
-  <link rel="stylesheet" href="{{site.baseurl}}/assets/css/styles.css">
-  <style>
-    .usa-banner__content{
-      display: hidden;
-    }
-  </style>
-</head>
\ No newline at end of file
+  <link rel="stylesheet" href="{{site.baseurl}}/assets/uswds/css/uswds.min.css" />  
+  {% asset index.scss %}
+  
+  <script src="{{site.baseurl}}/assets/uswds/js/uswds.min.js"></script>
```

Comment:
Was there a reason the js was moved from the footer to head? When located here, it blocks rendering which is why it is typically at the bottom. We can leave it here if we add the `defer` and/or `async` attributes to the script tag to not degrade performance.

Reply:
moved to footer.